tests/src/Simple/HelloWasm/Program.cs

```diff
@@ -1316,6 +1326,24 @@ public void Dispose()
     }
 }
 
+class FieldStatics
+{
+    static int X;
+    static int Y;
+    static string S1;
+    static string S2;
+
+    public bool TestGetSet()
+    {
+        X = 17;
```

Comment:
Could you also add a test that these are zero initialized before the assignments?
Just to make sure we're writing into locations that look right.